README.md

```diff
@@ -71,17 +85,27 @@ For example:
 ```
 
 ## Logging Programmatic Errors
-In order to log custom errors which will be visible in the platform, you can use `console.log("[LUMIGO_LOG] <YOUR_MESSAGE>");` from anywhere in your lambda code.
+With the tracer configured, simply call `console.log("[LUMIGO_LOG] <YOUR_MESSAGE>");` to cerate custom errors that are visible throughout the platform. This can be used anywhere in your Lambda code, and is included with the `@lumigo/tracer` package.
 
 ## Adding Execution Tags
-You can use `addExecutionTag` function to add an execution tag with a dynamic value.<br/>
-This value can be searched within the Lumigo platform.<br/>
-### Manual tracing
-Add `const lumigo = require('@lumigo/tracer')({ token: 'DEADBEEF' })`.<br/>
-Then use `lumigo.addExecutionTag('<key>', '<value>');` from anywhere in your lambda code.<br/>
-### Auto tracing
-Add `const lumigo = require('@lumigo/tracer');`.<br/>
-Then use `lumigo.addExecutionTag('<key>', '<value>');` from anywhere in your lambda code.<br/>
-### Limitation:
+You can add execution tags to a function with dynamic values using the parameter `addExecutionTag`.
+
+These tags will be searchable from within the Lumigo platform.
+
+### Adding tags for Manual tracing
+To add a tag to a manual trace statement:
+
+* Add `const lumigo = require('@lumigo/tracer')({ token: 'YOUR-TOKEN-HERE' })` to your code.
+* Add execution tags by using `lumigo.addExecutionTag('<key>', '<value>');` 
+
+### Adding tags for Auto tracing
+To add a tag to an automatically-traced function:
+
+* Add ``const lumigo = require('@lumigo/tracer')({ token: 'YOUR-TOKEN-HERE' })`` to the top of your handler's .js file
```

Comment:
When using auto tracing no need to define your token

Reply:
Ahh, got it! Removed the token from the code sample